config-controller/test/utils/utils.go

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
remove?

Reply:
This is use in `test/e2e/e2e_test.go`. Should we remove all the generated e2e tests?  I know the acs operator uses kuttl.  Should we standardize on that?

Reply:
I prefer the e2e_test like here, if you are interested in kuttl I suggest you talk to @porridge. 
I think in the scenario here, where the CRDs configure StackRox, kuttl might not be a good fit because it can only test on Kubernetes resources. If you want to do something programatically it gets complex.

Reply:
Actually just yesterday [someone suggested](https://github.com/kudobuilder/kuttl/issues/552) adding the ability to retrieve "actual" objects using arbitrary commands. Assuming we could simply translate central API responses to YAML, this would be a good fit.